db/ecf_schema.rb

```diff
@@ -0,0 +1,1474 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2024_02_06_170145) do
```

Comment:
Im wondering how will we keep this up to date with ECF going forward?

Reply:
I think this will just need to be a manual task to begin with; I don't think anything will break if it becomes out of date (unless a migration isn't backwards compatible, but most of ours will be). If we find we get errors in NPQ reg deployments because of it we could revisit automating it? we could pull the latest version in from ECF when we build the docker image